standard/docs/en/getting_started/releases_and_records.md

```diff
@@ -97,7 +97,7 @@ In some cases, releases will all come from the same backend system. In others, t
 
 In a complete OCDS implementation, each release would be published at its own persistent URL, and kept online permanently. 
 
-In some cases, a publisher might only have the data infrastructure to publish and iteratively update one release for each contracting process. If so, the release `id` and release `date` must be modified with each update. A suggested pattern for generating a unique release `id` is to append an incrementing number to the end of the `ocid`: for example, if the `ocid` is "ocds-a2ef3d01-1594121", then each sequential release `id` can be:
+In some cases, a publisher might only have the data infrastructure to publish and iteratively update one release for each contracting process. If so, the release `id` and release `date` need to be modified with each update. A suggested pattern for generating a unique release `id` is to append an incrementing number to the end of the `ocid`: for example, if the `ocid` is "ocds-a2ef3d01-1594121", then each sequential release `id` can be:
```

Comment:
In future we might want to update this paragraph to suggest alternative approaches (appending last modified date, or hashing the fields from the database) since I understand from some implementations (e.g. NSW) that appending a sequential number isn't possible

Reply:
I've made a note in our 1.1.5 examples spreadsheet.